src/content/1.7/scale/optimizations.md

```diff
@@ -4,23 +4,45 @@ alias:
 - 1.7/optimizations/
 ---
 
-# How to Optimize the performance of your PrestaShop
+# Optimize your PrestaShop performances
 
-In order to properly benchmark your shop, you need to check the settings of PHP, Apache and MySQL.
+Before making any change on your PrestaShop instance, it is highly recommended to perform some benchmarks.
 
-## 1) PHP Settings
+The idea behind this is to _know_ where you're coming from and make sure that your changes increase your performances, not the other way around.
```

Comment:
```suggestion
The idea behind this is to _know_ what your baseline performance is, so that you can make sure that your changes are increasing your shop's performance and not the other way around.
```